src/renderer/components/DeviceAction/index.js

```diff
@@ -219,6 +221,7 @@ const DeviceAction = <R, H, P>({
 const mapStateToProps = createStructuredSelector({
   reduxDevice: getCurrentDevice,
   preferredDeviceModel: preferredDeviceModelSelector,
+  lastSeenDevice: lastSeenDeviceSelector,
```

Comment:
i don't understand why deviceAction need them. to me we potentially just need to call a function to "refresh". and even that, i don't think we should do it in DeviceAction but just a side effect as we do for onboarding & manager.

Reply:
gotcha